webapp/interop_handler.go

```diff
@@ -5,73 +5,23 @@
 package webapp
 
 import (
-	"encoding/json"
 	"net/http"
-
-	"github.com/web-platform-tests/results-analysis/metrics"
-	"github.com/web-platform-tests/wpt.fyi/shared"
-	"google.golang.org/appengine"
-	"google.golang.org/appengine/datastore"
 )
 
 // interopHandler handles the view of test results broken down by the
 // number of browsers for which the test passes.
 func interopHandler(w http.ResponseWriter, r *http.Request) {
-	ctx := appengine.NewContext(r)
-	passRateType := metrics.GetDatastoreKindName(metrics.PassRateMetadata{})
-	query := datastore.NewQuery(passRateType).Order("-StartTime").Limit(1)
-
-	filters, err := shared.ParseTestRunFilterParams(r)
+	uiFilters, err := parseTestRunUIFilter(r)
```

Comment:
This is great! I refactored this handler a while ago to use `ParseTestRunFilterParams` and was planning for a follow-up refactor to use `LoadTestRuns`. It didn't occur to me we could and should move the fetching to the client. This is much more concise and consistent with the results handler!